src/lib/OpenEXRCore/internal_rle.c

```diff
@@ -42,7 +42,7 @@ internal_rle_compress (
         }
         else
         {
-            /* uncompressable */
+            /* uncompressible */
```

Comment:
Should be incompressible

Reply:
Added + 2 more related fixed [02fa272](https://github.com/AcademySoftwareFoundation/openexr/pull/1272/commits/02fa2723aa50f5a5fb1cb02a06d68f5b69bebb82)